bin/rad-diff

```diff
@@ -319,20 +308,20 @@
                (cmd-parse-failure (string-append "Invalid list option: " o)))) os)))
 
 (def /list-cmd
-  (fn [chain-name opts]
-    (/cmd-2-opts "list" chain-name opts valid-list-options list-options help)))
+  (fn [opts]
+    (/cmd-1-opts "list" opts valid-list-options list-options help)))
 
 (def args (get-args!))
 (match args
-  (/list-cmd 'chain-name 'options) (list-items chain-name options)
-  (/cmd-2 "propose" 'commit 'chain-name help) (new-diff! (make-name chain-name) commit)
-  (/cmd-2 "checkout" 'diff-no 'chain-name help) (checkout-diff! chain-name (read diff-no))
-  (/cmd-2 "show" 'diff-no 'chain-name help) (show-diff! chain-name (read diff-no))
-  (/cmd-2 "retract" 'diff-no 'chain-name help) (edit-diff-state! chain-name (read diff-no) :retracted)
-  (/cmd-2 "accept" 'diff-no 'chain-name help) (edit-diff-state! chain-name (read diff-no) :accepted)
-  (/cmd-2 "reject" 'diff-no 'chain-name help) (edit-diff-state! chain-name (read diff-no) :rejected)
-  ["comment" 'diff-no 'comment 'chain-name] (create-comment! chain-name (read diff-no) comment)
-  (/cmd-1 "create-chain" 'chain-name help) (new-diff-chain! chain-name)
+  (/list-cmd 'options) (list-items (chain-name) options)
```

Comment:
`chain-name` does not exist. You probably meant `base-name`. Better to rename `base-name` to `chain-name`, though.

Reply:
Fixed.